interview-questions.md

```diff
@@ -9,59 +9,63 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: From memory, this is a keyword in Javascript that is used to reference objects and access the values within objects/classes.
 
-  Researched answer:
+  Researched answer: After looking up this answer in the Syllabus, it appears I was on target with my initial answer.
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: From memory, React is a Javscript framework used for creating user interfaces. One would use React if they want to quickly spin up a project and value scaleability.
 
-  Researched answer:
+  Researched answer: React was made to create user interfaces. It can be used to build web and mobile pages. Everything in React is a componenet. It is highly scaleable and built by Facebook.
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: From memory, I believe the required lifecycle method in this case is the constructor.
 
-  Researched answer:
+  Researched answer: After looking up this answer in the syllabus, it appears I was on target with my initial answer.
```

Comment:
Since React is always looking to display something or _render_ the page, the lifecycle method is the render() method.